tensilelite/Tensile/TensileCreateLibrary/Run.py

```diff
@@ -476,9 +476,12 @@ def generateKernelHelperObjects(solutions: List[Solution], cxxCompiler: str, isa
                         buildMask.append(False)
                 if any(buildMask):
                     kho = initHelperKernelObjects(solution, kernelHelperType, cxxCompiler, isaInfoMap)
-                    kho = list(itertools.compress(kho, buildMask))
-                    if kho:
-                        khos.extend(kho)
+                    # Pair each kho with its name, then sort to match names order
```

Comment:
@ellosel There is a chance that buildMask is [A, B] while kho is [B, A]. Leading incorrect ko been pushed to khos.

Reply:
Done